src/app/features/cases/components/organisms/FineSummary/FineSummary.tsx

```diff
@@ -22,8 +23,8 @@ const FineSummary: React.FC<Props> = ({ id, date, state, hasInvoice = false }) =
     "Status": <Wrap>
       { state }
       { hasInvoice ?
-        <CircleIcon color="#00A03C" icon={ "Checkmark" } /> :
-        <CircleIcon color="#009DEC" icon={ "Ellipsis" } />
+        <CircleIcon color={ themeColor("support", "valid")({ theme: ascDefaultTheme }) } icon={ "Checkmark" } /> :
```

Comment:
What happens when you leave out `({ theme: ascDefaultTheme })`? Different colors?

Reply:
No, you'll get a TypeScript error

Reply:
If you know a way to use the actual theme vs. the default one. Let me know (or push).

Reply:
Ah right, probably because you pass them as a prop, instead of within a styled-component.
Never knew that :-).

Reply:
Yes, the theme gets passed in the CSS context, but not in TS. Have asked in Slack #frontend channel. If there is a better solution.

Reply:
Waiting for: https://github.com/Amsterdam/amsterdam-styled-components/issues/1108